client/lib/postal-code/index.jsx

```diff
@@ -85,15 +85,18 @@ export function tryToGuessPostalCodeFormat( postalCode, countryCode ) {
 		return postalCode;
 	}
 
-	if (
-		includes( countryCodeData.length, postalCode.length ) &&
-		postalCode.indexOf( countryCodeData.delimiter ) === -1
-	) {
+	const postalCodeWithoutDelimeters = replace( postalCode, /[ -]/g, '' );
```

Comment:
How about `\s` instead of ` `? In case somehow pastes a `\t` character or something even more magical.

Reply:
Done! :)